homeassistant/helpers/aiohttp_client.py

```diff
@@ -28,7 +30,8 @@ def async_get_clientsession(hass, verify_ssl=True):
         connector = _async_get_connector(hass, verify_ssl)
         clientsession = aiohttp.ClientSession(
             loop=hass.loop,
-            connector=connector
+            connector=connector,
+            headers={'User-Agent': SERVER_SOFTWARE}
```

Comment:
Can you use the [headers from aiohttp](https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/hdrs.py#L86)

```python
from aiohttp.hdrs import USER_AGENT
```